### DISCUSSION TITLE
Show Patchpanel in a Rack

### DISCUSSION THREAD

**Discussion by Alphanumerisch** [Molkobain/itop-extensions-documentation#1] [General]
How can I mount a patch panel in a rack and have it displayed? I don't want to create an extra network object. Is that possible?

**Comment by Molkobain** [comment]
Hello @NoValidUser , yes this can be done by following [this tutorial](https://molkobain.github.io/itop-extensions-documentation/molkobain-datacenter-view-extended/docs/tutorials/add-patch-panels.html). :)

**Comment by bushihehudo** [comment]
I haven't the extended datacenter extension only the datacenter view extension. It is possible to mount the patch panel in this version?

**Reply by Molkobain** [reply]
Hello @bushihehudo , unfortunately it's not possible, you can do this only with the extended version of the extension because it's using a configuration parameter brought by the extended version.
